docs/UserGuide.md

```diff
@@ -233,82 +252,75 @@ Examples:
-  Example 1: `findall [NAME]`: `findall John` will return contacts with gender attributes that contains the keyword like `John Doe` and `John The Builder`
-  Example 2: `findall [GENDER]`, `findall Male` will return contacts with gender attributes that contains the keyword like `Male`
-  Example 3: `findall [ADDRESS]`: `findall Blk 30` will return contacts with address attributes that contains the keyword like `Blk 16 Hello Drive` and `Blk Goodbye Drive`
-  Applicable for all the following attributes in a person's details: `[NAME] [GENDER] [PHONE_NUMBER] [EMAIL] [COMPANY] [LOCATION] [o/OCCUPATION] [JOBTITLE] [ADDRESS]`
+  Example 1: `findall NAME`: `findall John` will return contacts with gender attributes that contains the keyword like `John Doe` and `John The Builder`
+  Example 2: `findall ADDRESS`: `findall Blk 30` will return contacts with address attributes that contains the keyword like `Blk 16 Hello Drive` and `Blk Goodbye Drive`
+  Applicable for all the following attributes in a person's details: `NAME GENDER PHONE_NUMBER EMAIL COMPANY LOCATION OCCUPATION JOBTITLE ADDRESS`
 
-### Finding a contact tag: `findtxn`
+### Finding a contact and their transactions list: `findtxn`
 
-Search for a single contact and all txns related to this contact. The user must enter an existing user that is in the database and must match exactly the name in the database.
+Search for a single contact and all transactions related to this contact. The user must enter an existing user that is in the contact list and must match exactly the name in the contact list.
 
-Format: `findtxn [NAME]`
+Format: `findtxn NAME`
 
-- The search is case-insensitive. e.g `[John Doe]` will match `[john doe]` and all transactions with `[john doe]` as the owner in the transaction
-- The spaces before and after keywords does not matter. e.g. `[ Hans Bo  ]` will match `[Bo Hans]`
-  e.g. `findtxn [John Doe]` will return `John Doe` and all txns with the same owner name `John Doe`
+- The search is case-insensitive and exact-match. e.g `John Doe` will match `john doe` and all transactions with `john doe` as the owner in the transaction
+- The spaces before and after keywords does not matter. e.g. `Hans Bo ` will match `Bo Hans`
+  e.g. `findtxn John Doe` will return `John Doe` and all txns with the same owner name `John Doe`
 
 Examples:
 
-- `findtxn [John Doe]` - returns the contact with valid and associated transactions with the owner name `John Doe`
-
-### Deleting a person : `delete`
-
-Deletes the specified person from the address book.
+- `findtxn John Doe` - returns the contact with valid and associated transactions with the owner name `John Doe`
 
-Format: `delete INDEX`
+### Adding a transaction: `addtxn`
 
-- Deletes the person at the specified `INDEX`.
-- The index refers to the index number shown in the displayed person list.
-- The index **must be a positive integer** 1, 2, 3, …​
+Adds a transaction record to the contact list.
```

Comment:
Might be wrong, but should `contact list` be `transaction list`? Since deletetxn, listtxn, edittxn mentions `transaction list`.